argonaut/src/main/scala/io/finch/argonaut/Decoders.scala

```diff
@@ -1,6 +1,8 @@
 package io.finch.argonaut
 
 import argonaut.{CursorHistory, DecodeJson, Json}
+import com.twitter.finagle.netty3.ChannelBufferBuf
+import com.twitter.io.Charsets
```

Comment:
Please, use `StandardCharsets` from JDK. `c.t.i.Charsets` are deprecated in the most recent Finagle release.